api/src/libs/fulfillment.ts

```diff
@@ -0,0 +1,104 @@
+//
+// Copyright © 2020 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Created by Jason Leach on 2020-06-2.
+//
+
+import { logger } from '@bcgov/common-nodejs-utils';
+import { SUBJECTS } from '../constants';
+import DataManager from '../db';
+import { Contact } from '../db/model/contact';
+import { ProjectProfile } from '../db/model/profile';
+import shared from './shared';
+
+export interface Context {
+  something: any;
+}
+
+export const contextForProvisioning = async (profileId: number): Promise<any> => {
+
+  try {
+    const dm = new DataManager(shared.pgPool);
+    const { ProfileModel, ContactModel } = dm;
+    const profile: ProjectProfile = await ProfileModel.findById(profileId);
+
+    if (!profile) {
+      return;
+    }
+
+    const contacts: Contact[] = await ContactModel.findForProject(profileId);
+    const contact = contacts.filter(c => c.roleId === 2).pop();
+
+    if (!contact) {
+      return;
+    }
+
+    const context = {
+      profileId: profile.id,
+      displayName: profile.name,
+      namespaces: [
```

Comment:
fake data?

Reply:
Good catch. Its not meant to be. Fixing...

Reply:
Fixed.